ci/run.sh

```diff
@@ -3,6 +3,10 @@
 set -xeu
 set -o pipefail
 
+rdmd --compiler=${DC} ./tests/runner.d --compiler=${DC} -cov
```

Comment:
Could you move it below the `dub build -b unittest-cov` step? Otherwise we're introducing a big timeout here and we don't even know if the unittests build will compile.

Reply:
Will push an update after this current ci run.

Reply:
Ok all GREEN will squash the ci commits after moving the runner tests.

Reply:
Thanks @ferencdg single threaded is good for ci :-)

Reply:
enjoy :)